build/templates/session.py.mako

```diff
@@ -33,6 +33,41 @@ from ${module_name} import visatype
 from ${module_name} import ${c['file_name']}  # noqa: F401
 % endfor
 
+import pprint
+pp = pprint.PrettyPrinter(indent=4)
+
+
+class _TimedeltaConverter(object):
```

Comment:
We should try to put converters in their own files, it'd be cleaner.
It means that this template would need to figure out what to import, which can be done by iterating over parameters for all functions and sticking all needed converters in a set. Then expanding the import statements.

Reply:
Done.